src/docs/truffle/reference/truffle-commands.md

```diff
@@ -144,13 +135,11 @@ If you want an interactive console but want to use an existing blockchain, use `
 
 See the [Using the console](/docs/getting_started/console) section for more details.
```

Comment:
Not sure if this link works. May just be that I can't find it but when I try going to it I get a 404.

Reply:
Oh, good catch, I'm going to replace it with this one: https://trufflesuite.com/docs/truffle/getting-started/using-truffle-develop-and-the-console.html